src/main/java/com/softserveinc/ita/utils/runners/ElementsUtil.java

```diff
@@ -0,0 +1,51 @@
+package com.softserveinc.ita.utils.runners;
```

Comment:
why this class here?

Reply:
So, where he should be? If we was always put him in utils.runners before?

Reply:
probably it was a mistake, in runners package should be only runners, move into utils package